cirq/google/vis/README.md

```diff
@@ -0,0 +1,25 @@
+This directory contains convenience functions for visualizations.
```

Comment:
we have not tended to use readme in code base and instead prefered docs in our top level docs location.  My worry is not that this wouldn't be a good pattern, but that we are going to end up having duplicate locations for people to look for code.  So I would prefer for now that this ends up in the docs

Reply:
I found that docs in codes and README.md serve different purposes. Docs in codes describe what the immediate code does. README.md describes what the directory is for from a higher level. It's very useful for newcomers to get a quick picture of how codes fit together by reading the README.md of each directory.

Github automatically shows README.md when a user navigates to a directory. That helps a lot, too.

Reply:
The other problem is that this will become nearly unmaintainable for say cirq.ops.  We have not tended to use cirq heirarchy for organizational purposes (I'll admit that I like to do this, but early on we didn't do this and flatten everything to the top level of cirq.google.)

Reply:
I'm not advocating to have `README.md` everywhere. I think we can view this new directory as an experiment of that.

Flatten everything to the top is convenient for users of the library. Having `README.md` helps contributors / developers, or users that are curious.

When the directory grows large, it's natural to just document the top level functionality in `README.md`. At that point code examples can be moved to an `example/` subfolder.

Reply:
+1 with @dabacon that we should not have a `README.md` here and instead move this information to the `docs` directory. We should keep documentation in 2 locations; `docs` and comments in the source files to avoid documentation being "everywhere" so to speak.

Reply:
I've moved codes in `README.md` into an script in `examples/` subdirectory next to the figure it produces.